src/player.ts

```diff
@@ -405,4 +405,18 @@ export class Player extends NativeInstance<PlayerConfig> {
   castStop = () => {
     PlayerModule.castStop(this.nativeId);
   };
+
+  /**
```

Comment:
We don't have a description here. Please see the other methods that have a description and the return value documented.

Reply:
Address in cafb79f9ec5399959d523103108a7cf581495c07